proto/ethermint/evm/v1/tx.proto

```diff
@@ -158,3 +165,19 @@ message MsgEthereumTxResponse {
   // gas_used specifies how much gas was consumed by the transaction
   uint64 gas_used = 5;
 }
+
+// MsgUpdateParams is the Msg/UpdateParams request type for EVM parameters.
+message MsgUpdateParams {
+  option (cosmos.msg.v1.signer) = "authority";
```

Comment:
Havn't seen this option before. Does this achieve the same as https://github.com/evmos/ethermint/pull/1472/files#diff-998350afce90348a623164365b2d93781b5d99eb7ff0d4f661b42bc7eff19ec2R380?

Reply:
Yes I should probably delete the `GetSigners` function